api-put-object-multipart.go

```diff
@@ -141,7 +141,7 @@ func (c Client) putObjectMultipartNoStream(bucketName, objectName string, reader
 	}
 
 	// Verify if we uploaded all the data.
-	if size > 0 {
+	if size >= 0 {
```

Comment:
The reason this is done is multipart should also work for size == 0 independent of its upper layer usage.